pkg/ipamd/ipamd.go

```diff
@@ -1920,3 +1949,30 @@ func (c *IPAMContext) getPrefixesNeeded() int {
 	log.Debugf("ToAllocate: %d", toAllocate)
 	return toAllocate
 }
+
+func (c *IPAMContext) BroadcastMissingPermissionsEvent(nodeInitErr error) {
+	printErr := strings.Replace(nodeInitErr.Error(), "\n", "", 1)
+	log.Errorf("Failed to create pod, please verify AmazonEKS_CNI_Policy is attached to supplied IAM Role: {%v}", printErr)
```

Comment:
- This error implies the cluster operator must use a specific IAM policy. I think any equivalent policy would be valid too.
- I think `UnauthorizedOperation` could be the result of an explicit deny, so that error could message could be shown even when the IAM role has that permission attached.
- do we _always_ know that the principal running this code is an IAM role?

I think I'd prefer to know which action was found to be forbidden.

Reply:
@sftim - Yes it makes sense, we can log the action which was forbidden.

Reply:
Thank you. I've modified the alarm message. 
The error log shows the request-id for the unauthorized operation.